superset/assets/spec/javascripts/dashboard/fixtures/mockSliceEntities.js

```diff
@@ -37,3 +37,141 @@ export default {
   errorMessage: null,
   lastUpdated: 0,
 };
+
+export const sliceEntitiesForDashboard = {
+  slices: {
+    '127': {
```

Comment:
nit -- these don't need quotes / looks like prettier doesn't cover fixtures. nbd we can fix in another linting pass.